src/GallerySwiper.jsx

```diff
@@ -1009,9 +936,24 @@ class GallerySwiper extends Component {
             }
         });
 
+        let events;
+        if (!disableArrowKeys) {
+            events = (
+                <AttachHandler
+                    target={BASE_CLASS}
```

Comment:
@lucidNTR you just use a simple uid using a random number and add to to state when the component mounts.